src/views/GroupTable/GroupTable.tsx

```diff
@@ -49,6 +50,17 @@ const GroupTable: FC<Props> = props => {
 	const currentGroup = userGroups.find(
 		(x: UserGroupState) => x.groupId.toString() === tableView.selectedGroupId.toString()
 	);
+
+	let userTable = <UsersTable users={filteredUsersTable}
```

Comment:
prefer to write it inline (makes the HTML more readable), so it would look like:
```
<div className={classes.root}>
			<Typography className={classes.center} variant='h2'>
				{currentGroup ? currentGroup.displayName : null}
			</Typography>
			<UsersToolbar searchText={groups.searchText} onSearchInputChange={(text: string) => {
				dispatch(searchGroupTable(text))
			}} />
			<div className={classes.content}>
				<UsersTable users={filteredUsersTable}
							onUserClicked={(predictionId: number) => {
								history.push(`/prediction/${predictionId}`);
							}} />
				     { filteredUsersTable.length === 0 ?
               <Typography align='center' paragraph>Oops, no matching users</Typography>
               : 
              <UsersTable users={filteredUsersTable}
							      onUserClicked={(predictionId: number) => {
								          history.push(`/prediction/${predictionId}`);
							       }} />
         }
			</div>
		</div>
```

Reply:
did you mean declaring the UsersTable component only once inside the curly braces? because   according to this snippet, if the table is not empty then it would be shown twice..

Reply:
sorry - my bad, fixed it. it should be only once